Sources/Tqdm/Tqdm.swift

```diff
@@ -232,6 +232,7 @@ public class Tqdm {
             lastPrintN += Float(n) // for auto-display logic to work
         }
         self.n += Float(n)
+        self.n = min(self.n, self.total!)
```

Comment:
This change makes it behave differently from [the Python library](https://github.com/tqdm/tqdm). The original one allows you to increase counter even if it is larger than the total. Another issue is that you unwrap `self.total`, it may be `nil` and `nil` has a meaning for the API.

 Therefore, I think it is better to remove this line.

Reply:
Thanks for the response and sorry for delay. 

I will remove this one. I did not realize the objective was to keep the same behavior as the Python-tqdm even though some of the design choices might have been conceived differently.

Reply:
I have removed my edit but I might keep it in my repo as I need that behavior.